metadata-ingestion/src/datahub/ingestion/source/feast.py

```diff
@@ -91,6 +96,9 @@ class FeastRepositorySourceConfig(ConfigModel):
     environment: str = Field(
         default=DEFAULT_ENV, description="Environment to use when constructing URNs"
     )
+    owner_mappings: List[Dict[str, str]]  = Field(
```

Comment:
Let's also add 2 additional flags:

- `enable_owner_extraction`: BOOL - If this is disabled, then we NEVER try to map owners. If this is enabled, then owner_mappings is REQUIRED to extract ownership. (We can use something like this for the docs)
- `enable_tag_extraction`: BOOL - If this is disabled, then we NEVER try to extract tags.

Reply:
done